### PR TITLE
google analytics issue fix

### PR DESCRIPTION
**Pull request by bennpearson**
### What is the context of this PR?
Fixs #977 Google analytics events were behaving erratically so this fixes the issue.

### How to review 
- in terminal `set -x EQ_UA_ID UA-56892037-7`
- Select schema `census_household.json`
- navigate to question that doesn't have a label and click on `guidance`, `help and support`, `previous`.
- cause an error on the page
- check google analytics

### CODE REVIEW

File: app/templates/layouts/_base.html [hunk 1]
```diff
@@ -33,7 +33,18 @@
     <![endif]-->
     {% endblock styles -%}
     {% endblock head -%}
-    {{ analytics }}
+    <!-- Google Analytics -->
+    <script>
```
Comment (collisdigital) [inline app/templates/layouts/_base.html:37]: I wonder if we should put this into a separate template and include it here to keep it separate?

**Review by collisdigital — CHANGES_REQUESTED** [review]
You also need to remove everything under `app/analytics/` and the setup_analytics() function in `app/__init__.py` and calls to it.

File: app/assets/js/app/modules/analytics.js [hunk 2]
```diff
@@ -8,7 +8,9 @@ export default function initAnalytics() {
   }
 
   if (typeof window.ga !== 'undefined') {
-    trackEvent = window.ga
+    trackEvent = function(evt, data) {
```
Comment (hamishtaplin) [inline app/assets/js/app/modules/analytics.js]: If we're gonna change this then:

```javascript
trackEvent = (evt, data) => window.ga(evt, data)
```

is more concise

**Review by hamishtaplin — CHANGES_REQUESTED** [review]

**Review by hamishtaplin — APPROVED** [review]

**Review by collisdigital — COMMENTED** [review]
Looks good to me, needs the team to review it

**Review by collisdigital — APPROVED** [review]

**Pull request merged**